app/views/layouts/mailer.html.erb

```diff
@@ -15,10 +15,9 @@
         <div class="col-lg-1"></div>
         <div class="col-lg-10">
           <div class="row mt-2">
-            <div class="col-12 bg-black">
+            <div class="col-12 dark">
               <%= link_to root_url do %>
-                <%= image_tag "nav-brand-transparent-#{I18n.locale}.png", class: 'ax-center', width: '300px',
-                                                                          height: '300px' %>
```

Comment:
From what I can see its more common to have header/footer in emails to be centered. The layout doesn't have to match the application, just the colours. Can you recentre the logo and make the footer content centered.